templates/CRM/Contribute/Page/ContributionTotals.tpl

```diff
@@ -46,14 +46,10 @@
               <th class="left contriTotalRight"> &nbsp; {ts}# Completed{/ts} &ndash; {$contributionSummary.total.count}</th>
             </tr><tr>
               <th class="contriTotalLeft">{ts}Avg{/ts} &ndash; {$contributionSummary.total.avg}</th>
-              <th class="right"> &nbsp; {ts}Median{/ts} &ndash; {$contributionSummary.total.median}</th>
-              <th class="right contriTotalRight"> &nbsp; {ts}Mode{/ts} &ndash; {$contributionSummary.total.mode}</th>
```

Comment:
I guess you have to add that dumb `contriTotalRight` class to the only `th` left in this row, so it will have both `contriTotalRight` and `contriTotalLeft` classes.

Reply:
Yes please do this one too.

Reply:
@colemanw can you test something locally - I get the best results for a contact with more than one currency by removing the tpl handling for more than one currency ie

```
          {if $contributionSummary.total.amount}
            <th class="contriTotalLeft right">{ts}Total{/ts} &ndash; {$contributionSummary.total.amount}</th>
            <th class="right"> &nbsp; {ts}# Completed{/ts} &ndash; {$contributionSummary.total.count}</th>
            <th class="right contriTotalRight"> &nbsp; {ts}Avg{/ts} &ndash; {$contributionSummary.total.avg}</th>
          {/if}
```
and NO
```
{if $contributionSummary.total.currencyCount gt 1}
```

Might be something we can just strip out?

Reply:
![screenshot 2019-02-18 14 08 17](https://user-images.githubusercontent.com/336308/52922498-a6e26580-3386-11e9-908e-480fcf039dad.png)

Reply:
@colmanw - I just pushed that up